controllers/menuController.js

```diff
@@ -0,0 +1,95 @@
+const Menu = require("./../models/Menu");
+const jwt = require("jsonwebtoken");
+
+
+
+// Get all menu item
+
+const getAllMenu = async (req, res) => {
+    try {
+
+        const merchantId = req.merchantId;
+
```

Comment:
check that merchantId is is not undefined. If undefined, send the appropriate status code and a message saying, please login as a merchant or something along those lines,